Jenkinsfile

```diff
@@ -0,0 +1,88 @@
+pipeline {
+
+    agent { label "codejail-worker" }
+
+    options {
+        timeout(30)
+    }
+    stages {
+        stage('Install tox') {
+            steps {
+                withPythonEnv('System-CPython-2.7') {
+                    sh '''
+                    pip install -r requirements/tox.txt
+                    '''
+                }
+            }
+        }
+
+        stage('Run the tests for each sandbox') {
+            parallel{
+                stage('Run tests with python 2.7') {
+                    environment {
+                             CODEJAIL_TEST_USER = 'sandbox'
```

Comment:
Looks like the indentation got misaligned here.

Reply:
oops. Thanks